gadgets/trace_exec/test/integration/trace_exec_test.go

```diff
@@ -155,6 +167,9 @@ func TestTraceExec(t *testing.T) {
 					utils.NormalizeProc(&e.Proc)
 					utils.NormalizeInt(&e.Loginuid)
 					utils.NormalizeInt(&e.Sessionid)
+					utils.NormalizeInt(&e.DevMajor)
```

Comment:
The test fails because dev_major is 0. It seems that for overlayfs, a device major can be zero:
```bash
$ stat -c '%Hd,%Ld' /bin/sh  # on the host (ext4)
253,1
$ docker run -ti --rm fedora stat -c '%Hd,%Ld' /bin/sh  # in the container (overlayfs)
0,82
```

So I suggest to remove `utils.NormalizeInt(&e.DevMajor)` and add a comment to explain.

```suggestion
					// Don't use NormalizeInt() because 0 is a valid device major for overlayfs
					e.DevMajor = utils.NormalizedInt
```